mix.exs

```diff
@@ -14,7 +14,12 @@ defmodule Dictator.MixProject do
       package: package(),
       deps: deps(),
       name: "Dictator",
-      source_url: "https://github.com/subvisual/dictator"
+      source_url: "https://github.com/subvisual/dictator",
+      dialyzer: [
+        plt_add_apps: [:mix, :ex_unit],
+        ignore_warnings: ".dialyzer_ignore.exs",
+        flags: [:underspecs, :race_conditions]
```

Comment:
Why these flags specifically?

Reply:
honestly not sure. I've been copying them from project to project for a while now. I can no longer remember